### DISCUSSION TITLE
Version 8.2 Stable

### DISCUSSION THREAD

**Discussion by Arnova8G2** [Arnova8G2/Gcam_8.7.250.44#56] [Ideas]
Change logs:

**Even better stability**

- Fix slow motion, now videos is smooth
  *) fix green feeling use operation mode = normal
- Fix cinematic, now videos is smooth
- Add force video fps device ( camcorder.codec_device_fps ) 
- Fix astro timer
- Force astrotimelapse *Experimental
- Fixed many errors

------------------------------------------------
APK certificate fingerprints: 
SHA-1: e0be67031a86c4178239256f80d631d94451e263 
------------------------------------------------

APK file name: **arn.android.gcam**  
Gcam-8.7.250.build-8.2.230530.2037.apk 
 
APK file hashes: 
MD5: 95f79bcc22169e80b1733096ddff255b 
SHA-1: 5cc1431c76da2db9153f41f1137c37393a9e597a 

APK file name: **com.google.android.GoogleCamera**  
Gcam-8.7.250.build-8.2.230530.2044.apk 
 
APK file hashes: 
MD5: 1598dcce260915ac86cdfe6145b35cac 
SHA-1: 2ad02c8fb399cc2947ab967d5c2e01abba930008 

APK file name: **com.samsung.android.scan3d**  
Gcam-8.7.250.build-8.2.230530.2046.apk 
 
APK file hashes: 
MD5: 2bdd7e4f5c1f5159d16d89c5fa2dba8f 
SHA-1: c1d2b7b8163e8a5df2366eb18486f4f60cafb2a1 

APK file name: **com.ss.android.ugc.aweme**  
Gcam-8.7.250.build-8.2.230530.2048.apk 

APK file hashes: 
MD5: c256032268b1d25b29901fb71f5ef0bc 
SHA-1: a40b8cad57456449049f1f59ac24fb89d7a0c96c 

APK file name: **org.codeaurora.snapcam**  
Gcam-8.7.250.build-8.2.230530.2049.apk 
 
APK file hashes: 
MD5: 2be5d3ae22eda3fa94c1ab3c657aa9dc 
SHA-1: 53592ec0178d327c066746ea76aa4e1e54121314 

<hr /><em>This discussion was created from the release <a href='https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.2'>Version 8.2 Stable</a>.</em>

**Comment by myselflatu** [comment]
@Arnova8G2 social share crash is back

**Reply by Arnova8G2** [reply]
Fixed : https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.3

**Comment by myselflatu** [comment]
![Screenshot_2023-05-31-15-05-50-737_com miui bugreport](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40817763/947e236a-dbf7-403d-95a0-a4d31920c07e)
